docker-compose.yml

```diff
@@ -30,7 +30,9 @@ services:
     restart: on-failure
 
 volumes:
-  node-db:
+  node-mainnet-db:
+  node-testnet-db:
+  wallet-mainnet-db:
+  wallet-testnet-db:
```

Comment:
out of curiosity, did you try templating this with an ENV var too :thinking: ? 

```
node-${NETWORK}-db: 
wallet-${NETWORK}-db:
```

?

Reply:
I did, it doesn't work :(

```
ERROR: The Compose file './docker-compose.yml' is invalid because:
volumes value 'node-${NETWORK}-db', 'wallet-${NETWORK}-db' do not match any of the regexes: '^[a-zA-Z0-9._-]+$'
```